src/Arduino/EssaimRobots/EssaimRobots.ino

```diff
@@ -30,6 +30,7 @@
 // PololuBuzzer buzzer; // Comment when using Arduino MEGA
 
 DW_uncouping_controller_T uncoupling_controller_T;
+B_Robot_controller_decouplant_T Robot_controller;
```

Comment:
- We said to use English names, so rename "decouplan" to "uncoupling". 
- According to the previous comment on the variable names of block signals and states, I suggest renaming these two to "uncoupling_controller_states_T" and "uncoupling_controller_signals_T" or something like this.